apps/graphql/src/definitions/Build.js

```diff
@@ -15,12 +18,24 @@ export const typeDefs = gql`
     aborted
   }
 
+  type BuildStats {
+    createdScreenshotCount: Int!
+    updatedScreenshotCount: Int!
+    passingScreenshotCount: Int!
+  }
+
   type Build {
     id: ID!
     createdAt: DateTime!
     updatedAt: DateTime!
     "The screenshot diffs between the base screenshot bucket of the compare screenshot bucket"
-    screenshotDiffs: [ScreenshotDiff!]!
+    screenshotDiffs(
+      filterPassing: Boolean
+      offset: Int!
+      limit: Int!
+    ): ScreenshotDiffResult!
+    "The passing screenshot diffs"
+    passingScreenshotDiffs(offset: Int!, limit: Int!): ScreenshotDiffResult!
```

Comment:
```suggestion
    screenshotDiffs(
      where: ScreenshotDiffWhere
      offset: Int!
      limit: Int!
    ): ScreenshotDiffResult!
```

```gql
input ScreenshotDiffWhere {
  passing: Boolean
}
```

Using a filter like that you can handle all the cases without duplicating anything.